package/gargoyle/files/www/js/basic.js

```diff
@@ -661,7 +674,21 @@ function saveChanges()
 			uci.set("network", "lan", "netmask", document.getElementById("bridge_mask").value);
 			uci.set("network", "lan", "gateway", document.getElementById("bridge_gateway").value);
 			uci.set("network", "lan", "dns",     document.getElementById("bridge_gateway").value);
-
+			
+			//setup new bridge network
+			if( getSelectedValue("bridge_mode") == "client_bridge")
+			{
+				preCommands = preCommands + "\nuci set network.bridgecfg=interface\nuci set network.wwan=interface\n";
+				uci.set("network", "bridgecfg", "proto", "relay");
+				uci.set("network", "bridgecfg", "network", "lan wwan");
+				
+				uci.set("network", "wwan", "proto", "dhcp");
+				
+				uci.set("dhcp", "lan", "ignore", "1");
+				
+				uci.set("firewall", "lan", "network", "lan wwan");
```

Comment:
this line is no good. should be something like
```
uci.set("firewall", "@zone[0]", "network", "lan wwan");
```

Need to test whether @zone[0] works all the time, which i suspect it doesn't. May need a better way of identifying the correct config section.